src/Symfony/Component/Finder/Finder.php

```diff
@@ -460,6 +461,18 @@ public function sortByAccessedTime()
         return $this;
     }
 
+    /**
+     * Reverse the sorting.
+     *
+     * @return $this
+     */
+    public function reverseSorting(bool $reverseSorting = true)
```

Comment:
Just asking: why allow passing an argument here instead of doing this:

```php
public function reverseSorting()
{
    $this->reverseSorting = true;

    return $this;
}
```

Reply:
Because there is no way to reset a finder, so if you want to use it many time, but with different options, it's better to have a "real" setter

Reply:
but that's inconsistent with most other APIs, which cannot be reset either.

If you really want to have a bunch of common config for the Finder, and then some changes, the solution is to apply all the common config, and then clone the object before applying each variant

Reply:
Fair enough ; I fixed the code